packages/edit-site/src/components/welcome-guide/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as interfaceStore } from '@wordpress/interface';
+
+/**
+ * Internal dependencies
+ */
+import WelcomeGuideEditor from './editor';
+import WelcomeGuideStyles from './styles';
+import { store as editSiteStore } from '../../store';
+
+export default function WelcomeGuide() {
+	const { isActive, isStylesOpen } = useSelect( ( select ) => {
+		const sidebar = select( interfaceStore ).getActiveComplementaryArea(
+			editSiteStore.name
+		);
+		const isStylesSidebar = sidebar === 'edit-site/global-styles';
+		const feature = isStylesSidebar ? 'welcomeGuideStyles' : 'welcomeGuide';
```

Comment:
@Mamaduka I noticed that if you go into 'Styles' first (having never seen either welcome guide), the 'Welcome to Styles' welcome guide displays as expected.

You might then do some editing with the styles sidebar still open.

But then when you close the styles sidebar you also get the 'Edit your site' welcome guide, which feels out of place.

Not sure what the right solution would be here. 🤔 

Maybe the 'Edit your site' welcome guide shouldn't ever show when visiting the Styles link. Or maybe when visiting the Styles link, a Welcome guide that combines both 'Edit your site' and 'Welcome to Styles' should be shown.

Reply:
@talldan, I agree it's a little confusing.

I can use the query argument `styles=open` for this logic. Or maybe we should combine welcome guides since it's the same screen, just different entry points.

We can switch back to separate Welcome Guide modals once the "Global Styles" has a dedicated screen.

@jameskoster, @kellychoffman, what do you think?